middleware/index.js

```diff
@@ -53,6 +55,7 @@ middleware.postPermission = async function(req, res, next) {
     return res.redirect('back');
 }
 
+// check if user is allowed to Announcements
```

Comment:
Suggestion - grammatical fix ("Check if user is allowed to create Announcements")